artifacts/saml2020.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
These keys and certs are intentionally checked in. They are copied from their previous home in `keys.example` and `certs.example`.